documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -372,7 +507,46 @@ For an _AnnotatedRelationshipElement_ named “CurrentFlowFrom”, with an annot
 }
 ----
 
-For an _Entity_ named “MySubAssetEntity”, the payload is minimized to the following:
+If the _RelationshipElementAnnotation_ itself is the subject for ValueOnly the serialization is minimized to the following:
+
+[source,json,linenums]
+----
+{
+    "first": {
+      "type": "ModelReference",
+      "keys": [
+        {
+          "type": "Submodel",
+          "value": "http://customer.com/demo/aas/1/1/1234859590"
+        },
+        {
+          "type": "Property",
+          "value": "PlusPole"
+        }
+      ]
+    },
+    "second": {
+      "type": "ModelReference",
+      "keys": [
+        {
+          "type": "Submodel",
+          "value": "http://customer.com/demo/aas/1/0/1234859123490"
+        },
+        {
+          "type": "Property",
+          "value": "MinusPole"
+        }
+      ]
+    },
+    "annotation": [
```

Comment:
This maybe needs an explanation...I that the annotation points to a SubmodelElementCollection that, in-turn, contains a single Property of type String with value "TechnicalCurrentFlowDirection". 
Not sure if the original model, i.e. not serialized with content=value but with content=normal is in the document so the reader can understand this.

Reply:
But the example states that the annotation is a Property, not a Collection... the property is not part of a collection. But you are right: annotations is treated like SubmodelElementCollection w.r.t. its serialization. This text could be added.